sci-electronics/librepcb/librepcb-1.1.0.ebuild

```
@@ -0,0 +1,75 @@
+# Copyright 2009-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake virtualx xdg
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="https://librepcb.org/"
+SRC_URI="https://download.librepcb.org/releases/${PV}/${P}-source.zip"
+
+S="${WORKDIR}/${PN}-${PV/_/-}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="opencascade qt6"
+
+BDEPEND="
+	app-arch/unzip
+	!qt6? ( dev-qt/linguist-tools:5 )
+	qt6? ( dev-qt/qttools:6[linguist] )"
+
+RDEPEND="
+	dev-cpp/muParser:=
+	!qt6? (  dev-libs/quazip:=[qt5(+)]
+			dev-qt/qtconcurrent:5
+			dev-qt/qtcore:5
+			dev-qt/qtgui:5
+			dev-qt/qtnetwork:5[ssl]
+			dev-qt/qtopengl:5
+			dev-qt/qtprintsupport:5
+			dev-qt/qtquickcontrols2:5
+			dev-qt/qtsql:5
+			dev-qt/qtsvg:5
+			dev-qt/qtwidgets:5
+			dev-qt/qtxml:5
+	)
+	qt6? (  dev-libs/quazip:=[qt6(+)]
```

```suggestion
	qt6? (  dev-libs/quazip:=[qt6(-)]
```

quazip-1.2 has qt5 support but not qt6 support. Meaning the qt5 assumption is correct, but not the qt6 assumption.

---

thank you, fixed